04-Games/src/js/games/FallingStones.js

```diff
@@ -0,0 +1,240 @@
+import { GameTemplate } from "./GameTemplate.js"
+import { GameObject, MovableGameObject, Ball } from "../GameObject.js";
+
+export class FallingStones extends GameTemplate {
+
+    start() {
+        this.player = new MovableGameObject(175, 430, 50, 50, "#6bd26b", 0, 0);
+        this.gameOver = false;
+        this.points = 0;
+        this.life = 5;
+        this.bullets = [];
+        this.stones = [];
+        this.stonecount = 0;
+        this.shotcd = 30;
+        this.colors = {death:"#180b1d", health:"#c470b3", bonus:"#ba9714", strong:"#db0303"};
+    }
+
+    
+    bindControls() {
+        this.inputBinding = {  
+            "left": () => this.player.x += -(18+this.bonusPlayerSpeed),
+            "right": () => this.player.x += (18+this.bonusPlayerSpeed),
+            "up": () => this.create_balls(),
+        };
+    }
+
+    create_balls() {
+        if (this.shotcd >= 20) {
+            var bcolor;
```

Comment:
```suggestion
            let bcolor;
```
dann funktioniert natürlich der rest der funktion nicht mehr weil die Variable sauber gescoped ist und nicht einen auf global macht.